### PR TITLE
Remove unneeded routes

### LINKED ISSUE #4: Fix the routes names
Usually, you do not write in the route the name of the method. This is because the HTTP method (GET, PUT, POST, DELETE) already tells you that.

https://github.com/Gustavoggsb/price-search-api/blob/be241cd70c744c810638db29bf33e3b1776cf04a/src/products/products.controller.ts#L17

This should be: `:id` for example. The same for the others.

### PR DESCRIPTION
**Pull request by gustavoggsb**
Fixes #4

### CODE REVIEW

File: products.http [hunk 1]
```diff
@@ -24,8 +24,8 @@ Content-Type: application/json
 
 ###
 
-GET {{baseUrl}}/products/findById/{{productId}}
+GET {{baseUrl}}/products/{{productId}}
 
 ###
 
-Delete {{baseUrl}}/products/delete/{{productId}}
+Delete {{baseUrl}}/products/{{productId}}
```
Comment (felipecrs) [inline products.http]:
```suggestion
DELETE {{baseUrl}}/products/{{productId}}
```

File: src/products/products.model.ts [hunk 2]
```diff
@@ -17,9 +17,3 @@ export class Product {
 
 export const ProductSchema = SchemaFactory.createForClass(Product);
 
-export interface Product {
-  id: string;
-  name: string;
-  weight: number;
-  imageUrl: string;
-}
```
Comment (felipecrs) [inline src/products/products.model.ts]: Again? You did that yesterday as far as I remember. I guess your branch is not up-to-date. Try:

```
git pull origin master
```

**Review by felipecrs — CHANGES_REQUESTED** [review]
Not a good PR title. The PR title must be very clear about its purpose. How about:

_Remove unneeded routes_

The _Fixes #5_ should be in the PR description.

**Comment by Gustavoggsb** [comment]
> Not a good PR title. The PR title must be very clear about its purpose. How about:
> 
> _Remove unneeded routes_
> 
> The _Fixes #5_ should be in the PR description.

**Review by felipecrs — CHANGES_REQUESTED** [review]
Missing corrections from the previous review.

**Review by felipecrs — APPROVED** [review]

**Pull request merged**